nucliadb_telemetry/nucliadb_telemetry/fastapi/__init__.py

```diff
@@ -88,3 +96,6 @@ def instrument_app(
         # add last to catch all exceptions
         # `add_middleware` always adds to the beginning of the middleware list
         app.add_middleware(SentryAsgiMiddleware)
+
+    if trace_id_header is not None:
+        app.add_middleware(CaptureTraceIdMiddleware, response_header=trace_id_header)
```

Comment:
We have to be careful with middleware order. `OpenTelemetryMiddleware` starts the span you are getting in `CaptureTraceIdMiddleware`. Could you add a comment to note this?

Reply:
agree, done!